Core/Utilities/ElasticSearchDB.py

```diff
@@ -21,6 +21,8 @@
 from DIRAC.FrameworkSystem.Client.BundleDeliveryClient import BundleDeliveryClient
 
 
+sLog = gLogger.getSubLogger(__name__)
```

Comment:
If you are using this sublogger (why not, I like it) then please apply to all the module. It's just replacing `gLogger.` with `sLog.`.

Just one minor comment: we normally call it `LOG`:
```suggestion
LOG = gLogger.getSubLogger(__name__)
```

Reply:
Which is, BTW, what's done in the upper branches.

Reply:
@atsareg requested that we move to `sLog` https://github.com/DIRACGrid/DIRAC/pull/4304#discussion_r334868828, and we did https://github.com/DIRACGrid/DIRAC/commit/033b527fc429c75f0834fc1c9223211f61899e8b

Reply:
Ah, OK. Good then.

Reply:
I have repleaced all gLogger with sLog in recent commit.

Reply:
we did move to `sLog`, totally ignoring my comment and our own rules :D

Reply:
more seriously, if we want to make t hat a (good) habit, let's put it somewhere in the developer guide once for all

Reply:
@atsareg , Is there anymore stuff that needs to be done ? 
If not then we can merge this.